google/cloud/bigtable/tools/convert_acceptance_tests.py

```diff
@@ -82,8 +92,12 @@ def print_test(t):
     o += "\n"
 
     ok = True
-    if t["results"]:
-        ok = not any([r["error"] for r in t["results"]])
+    if ('results' in t):
+       	try:
+	    if ('error' in t["results"]):
```

Comment:
would this exception handling create any side effect on generated code?

Reply:
> would this exception handling create any side effect on generated code?

Exception handling can be removed as its included as part of debugging code. Code works well after removing try-except also.

Reply:
so lets remove it,